app/views/shared/partials/_footer-global.html.erb

```diff
@@ -0,0 +1,5 @@
+  <footer class="footer__global">
+    <small>An exploration by Viget’s intern class of 2017.
+  	&copy; Ground Rules 2017. All rights reserved.</small>
+    <a href="http://www.viget.com"><%= image_tag("vigetLogo.png") %></a>
```

Comment:
Might as well throw out the ruby code altogether!
`<img src = "vigetLogo.png" alt = "">` should suffice.